### DISCUSSION TITLE
Auto account?

### DISCUSSION THREAD

**Discussion by Dena-go-tech** [requarks/wiki#6918] [Help / Questions]
We have a wiki that has comments, but for the comments to be posted correctly, users need to have an account, and it’s difficult for me to create them manually.
I would like to know how to make it so that users can create accounts themselves, preferably so that users can create accounts through discord.

I would be grateful if you answer.

Your Dena_

**Comment by fnafox** [comment]
You can use Discord as an authentication strategy:
https://docs.requarks.io/auth/discord
